src/tap/src/tapcell.cpp

```diff
@@ -185,9 +185,10 @@ int Tapcell::placeTapcells(odb::dbMaster* tapcell_master,
   }
 
   const odb::Rect row_bb = row->getBBox();
-
+  odb::Rect query_box;
+  row_bb.bloat(-1, query_box);
   std::set<odb::dbInst*> row_insts(
-      fixed_instances.qbegin(boost::geometry::index::covered_by(row_bb)),
+      fixed_instances.qbegin(boost::geometry::index::intersects(query_box)),
```

Comment:
Could you use overlap rather than bloat(-1) & intersects?

Reply:
I did that [before](https://github.com/The-OpenROAD-Project/OpenROAD/pull/7266/commits/75dccb5877d2a855215bc58ebf3bd50fbc43c637), but I found that bgi::overlaps returns false if the one of the shapes completely contains the other shape.